R/calcEU_ReferenceScenario.R

```diff
@@ -12,7 +12,7 @@ calcEU_ReferenceScenario <- function() {
 
   weights <- x
   weights[, , ] <- NA
-  weights[, , "Price|Secondary Energy|Electricity (US$2005/GJ)"] <- 1
+  weights[, , "Price|Secondary Energy|Electricity (EUR2013/GJ)"] <- 1
```

Comment:
@fbenke-pik is this bugfix correct and shall we include it here or separately?

Reply:
Yes, this is correct. Thank you for fixing.